### PR TITLE
add blogpost

### PR DESCRIPTION
**Pull request by arivanandan**
[Preview](https://deploy-preview-50--cbweb.netlify.com/make-your-react-native-app-lighter-and-faster)

### CODE REVIEW

**Comment by arivanandan** [comment]
[Preview](https://deploy-preview-50--cbweb.netlify.com/make-your-react-native-app-lighter-and-faster)

File: posts/make-your-react-native-app-lighter-and-faster/2019-11-14-make-your-react-native-app-lighter-and-faster.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+link: /make-your-react-native-app-lighter-and-faster
+category:
+- Technology
+tags:
+- react-native
+- mobile-app-development
+author: Arivanandan
+---
+
+# Make your React Native app lighter and faster
```
Comment (sachinmjadhav) [inline posts/make-your-react-native-app-lighter-and-faster/2019-11-14-make-your-react-native-app-lighter-and-faster.md]: You can delete this line. Blog title will be generated from the `frontmatter`. Currently it's displaying two titles.

**Comment by sanandnarayan** [comment]
@sachinmjadhav can you work with @arivanandan and add some images to the post. There is too much text

**Comment by arivanandan** [comment]
Closing because I feel it doesn't have enough content.

**Pull request closed without merging**